docs/Guides/oauth2.md

```diff
@@ -183,6 +183,7 @@ The amount of information returned is determined by the scopes which your applic
 {
     "sub": "U-b10e6de6-ee43-429c-ac26-b3cb81ef6f5f",
     "email": "mary@example.com",
+    "email_verified": true,

```

Comment:
Please check to see if there are any other fields on the model that are missing. I noticed `legacy_user_id` earlier.

Reply:
Will do 🖖

Reply:
Done